### DISCUSSION TITLE
q群搜不到了 大佬

### DISCUSSION THREAD

**Discussion by 420348264** [shanmiteko/LotteryAutoScript#178] [General]
![image](https://user-images.githubusercontent.com/8722979/184533065-5eb7faf2-d27d-42b6-b5c9-a09e421d024f.png)

**Comment by shanmiteko** [comment]
参考 #173 

已开

**Reply by 420348264** [reply]
安排上了

**Reply by StudyingHUYANG** [reply]
大佬，我也想进群问问
![image](https://user-images.githubusercontent.com/50046567/187424486-aa7a3b04-dcb1-41f4-9b57-5117bfabdad7.png)

**Reply by shanmiteko** [reply]
> 大佬，我也想进群问问
> ![image](https://user-images.githubusercontent.com/50046567/187424486-aa7a3b04-dcb1-41f4-9b57-5117bfabdad7.png)
> 

已开

**Reply by StudyingHUYANG** [reply]
大佬，分区这个我直接设置了1，现在可以运行，但是不太清楚为什么，可能还是需要去群里问问，添麻烦了

**Comment by StudyingHUYANG** [comment]
群号是1078777380
这个吗

**Reply by shanmiteko** [reply]
还没进吗，开群后有个人进群，我就又关掉了

**Comment by StudyingHUYANG** [comment]
是的是的，我不知道是那个群号




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2022年8月31日(星期三) 下午3:36
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [shanmiteko/LotteryAutoScript] q群搜不到了 大佬 (Discussion #178)





 
还没进吗，开群后有个人进群，我就又关掉了

**Reply by StudyingHUYANG** [reply]
需要大佬给透露下群号

**Reply by shanmiteko** [reply]
> 需要大佬给透露下群号
> 

就是公告那个

**Reply by StudyingHUYANG** [reply]
我没看到，如果大佬方便的话，发我一下可以吗，1004846422@qq.com

**Comment by StudyingHUYANG** [comment]
已申请进入1078开头那个，但是不知道暗号




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2022年8月31日(星期三) 下午4:07
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [shanmiteko/LotteryAutoScript] q群搜不到了 大佬 (Discussion #178)





  
需要大佬给透露下群号
  
就是公告那个
